foundations/the_front_end/project_html_css.md

```diff
@@ -46,6 +46,25 @@ If you have entered `git push origin main` and typed out your username and passw
 
 Okay, that's enough Git for the moment -- time to actually build stuff!
 
+### Warning About Looking at Other People's Finished Project Code Before You have Completed Your Own
+
+All of our projects have a "Student Solutions" section at the bottom for people to post their completed code and live websites for the projects. These are here for you to compare your **completed** code to and see what other routes people have chosen to take with their project code and designs. 
+
+We know it may be tempting to look at the solutions when you get stuck or don't know where to start, but **don't** do it! Here are some reasons why:
+
+- All of the submitted projects are other *student* solutions. This means they are written by other people who are also still learning and may not necessarily be good examples to go off of.
+- Looking at solutions robs you of the chance to develop your problem solving and researching skills.
+- There are a ton of different ways to get the same end result, it is not "one size fits all".
+- In the real world, there will not always "solutions" for you to look at and you will have to come up with your own.
```

Comment:
...there will not always **be** 

Grammar error. 

This bullet point could also afford to be under the one for "Looking at solutions"